### PR TITLE
Ensure column is pinned to 0

### PR DESCRIPTION
**Pull request by karthiknadig**
Fix for https://github.com/microsoft/vscode-flake8/issues/141

We have tests to validate standard flake8. We need this change to ensure some of the plugins don't fail. Authors of some plugins use 0 based column index when flake8 themselves use 1 based column index. From the LSP server we can;t differentiate which plugin generated the error, so we masure that the column value does not become negative.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
     "python.testing.cwd": "${workspaceFolder}",
-    "python.formatting.provider": "black",
-    "editor.formatOnSave": true
+    "python.formatting.provider": "none",
```
Comment (rzhao271) [inline .vscode/settings.json:17]: Is this change intentional?

Comment (karthiknadig) [reply .vscode/settings.json:17]: yes
